examples/fit_ODNP_data.py

```diff
@@ -61,25 +82,35 @@
     fl.next(r"$R_{1}(p)$")
     fl.plot(R1p, "o", label="Experimental Data")
     # }}}
-    # {{{ fit krho inverse with two degrees of freedom and then apply to fit R1p
-    # {{{load in T100 dataset
+    # {{{ fit kᵨ⁻¹ with two degrees of freedom (to a straight line) and then
+    # apply to fit R1p
     T10_p = loadmat(
         search_filename(
-            T100_info['filename'], exp_type=T100_info['data_dir'], unique=True
+            T100_info["filename"], exp_type=T100_info["data_dir"], unique=True
         )
     )["a"][0, :]
-    R10_p = R1p.fromaxis("power").eval_poly(T10_p, "power") ** -1
-    # }}}
+    R10_p = 1 / (R1p.fromaxis("power").eval_poly(T10_p, "power"))
+    powers_fine = nddata(
+        r_[0 : R1p.getaxis("power") : 300j], "p"
+    )  # PR COMMENT: related to my comments below
     krho_inv = Ep.get_prop("acq_params")["concentration"] / (R1p - R10_p)
     krho_inv_coeff = krho_inv.polyfit("power", order=1)
-    krho_inv_fine = R1p.fromaxis("power").eval_poly(krho_inv_coeff, "power")
+    krho_inv_fine = R1p.fromaxis("power").eval_poly(
+        krho_inv_coeff, "power"
+    )  # PR COMMENT: why is this called "fine" it's only evaluated at the powers present in the R1p data, which is not that many
     M0, A, phalf, p = symbols("M0 A phalf power", real=True)
     R1p_expression = (T10_p[0] + T10_p[1] * p) ** -1 + (
         Ep.get_prop("acq_params")["concentration"]
         / (krho_inv_coeff[0] + krho_inv_coeff[1] * p)
     )
-    R1p_fit = lambdify(p,R1p_expression)
-    fl.plot(R1p_fit(R1p.fromaxis('power')), ls=":", color="k", label="Fit", alpha=0.5)
+    R1p_fit = lambdify(p, R1p_expression)
+    fl.plot(
+        R1p_fit(R1p.fromaxis("power")),
+        ls=":",  # PR COMMENT: again, why are you plotting this as :, since it's only evaluated at the powers of interest -- maybe you want to construct a new axis?  Why not just plot as a solid line, anyways?
```

Comment:
I was wasting time jumping between browser and vim, so I did this instead.

All of these need to be dealt with and removed before finalizing, please (you can also just search for PR COMMENT) in the code.

They should all be pretty straightforward.